bin/main

```diff
@@ -1,2 +1,36 @@
 #!/usr/bin/env ruby
-puts 'Hello world !'
+puts 'Welcome to the Tic-Tac_Toe game!'
```

Comment:
NB: This is not much of an issue but I just want to point out that the requirement was to have the game output `Hello world` in this milestone so you didn't have to make this change... yet. :wink: